@here/harp-mapview/lib/Utils.ts

```diff
@@ -162,32 +179,42 @@ export namespace MapViewUtils {
         const groundDistance = calculateDistanceToGroundFromZoomLevel(mapView, zoomLevel);
         const cameraHeight = groundDistance + (elevation ?? 0);
 
-        if (mapView.geoMaxBounds && worldTarget) {
-            const { distance } = constrainTargetAndDistanceToViewBounds(
-                worldTarget,
+        if (mapView.geoMaxBounds) {
+            // If map view has maximum bounds set, constrain camera target and distance to ensure
+            // they remain within bounds.
+
+            // tslint:disable-next-line: deprecation
+            const oldTarget = getTargetAndDistance(projection, camera, elevationProvider).target;
```

Comment:
is this different than the current `MapView.target` projected to world space?

Reply:
yes, fixed